docs/content/documentation/systems/Dampers/tensor_mechanics/ElementJacobianDamper.md

```diff
@@ -0,0 +1,11 @@
+# ElementJacobianDamper
+!description /Dampers/ElementJacobianDamper
+
+This damper limits the change in the Jacobians of elements. The damper becomes active if the relative change in Jacobian of any element exceeds the user defined maximum.
+
+!parameters /Dampers/ElementJacobianDamper
+
+!inputfiles /Dampers/ElementJacobianDamper
+moose/modules/tensor_mechanics/tests/jacobian_damper/cube_load.i
```

Comment:
@dschwen I thought you could put an example input file of the use of the class, but maybe that section is for some other purpose? Are there instructions somewhere for how these files are supposed to look?